src/lib/Save.js

```diff
@@ -0,0 +1,32 @@
+export default class Save {
+  constructor(logs) {
+    this._logs = logs;
+  }
+
+  downloadFile(blob, filename) {
+    const url = URL.createObjectURL(blob);
+
+    function listener(result) {
+      if (result.state.current === 'complete') {
+        setTimeout(() => {
+          URL.revokeObjectURL(url);
+        }, 100);
```

Comment:
@atiqueahmedziad Why are we using a setTimeout here? once we receive the onChanged event for the completed download can't we just immediately revoke that blob url?

Reply:
I saw somewhere in the internet that setTimeout is recommended for Firefox. But here it totally make sense to revoke that blob url as the download is completed. I will update it. :-)